File: ISSUE-67/SOLUTION-8/wpc67.c
#include   <stdio.h>

     int        gint_target = 9808;
     int        gint_pages;

     int        gint_loop;
     int        gint_try;

     int        gint_start;

     char gchar_results[100];

void main(void)
{
     gint_start = 1;

     do
           {
           gint_loop = gint_start;
           gint_try = 0;
           gint_pages = 0;

           while(1)
                {
                gint_try += gint_loop;
                gint_pages++;

                if(gint_try >= gint_target)
                     {
                     break;
                     }

                gint_loop++;
                }

           if(gint_try == gint_target)
                {
                break;
                }

           gint_start++;
           } while(gint_start < gint_target);

     sprintf(gchar_results, "There are %d pages missing starting at page %d\r\n", gint_pages, gint_start);

     while(gint_try != gint_loop);
}

File: ISSUE-67/SOLUTION-3/Wpc67.java

public class Wpc67 {

	private static final int TOTAL = 9808;
	
	public static void main(final String[] args) {
		for (int m = 1; m < TOTAL; ++m) {
			for (int n = m + 1; n < TOTAL; ++n) {
				if (n * (n+1) - m * (m+1) == 2*TOTAL) {
					System.out.println("Missing pages are numbered from " + (m+1) + " to " + n);
				}
			}
		}
	}
}
